src/libs/actions/OnyxDerived/configs/conciergeChatReportID.ts

```diff
@@ -0,0 +1,29 @@
+import {isThread} from '@libs/ReportUtils';
+import createOnyxDerivedValueConfig from '@userActions/OnyxDerived/createOnyxDerivedValueConfig';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+export default createOnyxDerivedValueConfig({
+    key: ONYXKEYS.DERIVED.CONCIERGE_CHAT_REPORT_ID,
+    dependencies: [ONYXKEYS.COLLECTION.REPORT, ONYXKEYS.CONCIERGE_REPORT_ID],
+    compute: ([reports, conciergeChatReportID]) => {
```

Comment:
now that we have current value available as a second parameter, we can return it immediately as I believe concierge chat report ID cannot change
```suggestion
    compute: ([reports, conciergeChatReportID], currentValue) => {
    	if (currentValue) {
    		return currentValue;
    	}
    }
```

Reply:
good suggestion, fixed!